libs/coin-bitcoin/src/mockBtc.ts

```diff
@@ -36,10 +37,30 @@ class MockBtc extends Btc {
     }
   }
 
-  // eslint-disable-next-line class-methods-use-this,@typescript-eslint/no-unused-vars
-  async createPaymentTransaction(arg: CreateTransactionArg) {
+  async createPaymentTransaction(arg: CreateTransaction) {
     return hash(arg);
   }
+
```

Comment:
This mock class has "getWalletXpub", "SignMessage", "SplitTransaction" because you inherit the "BitcoinSigner" interface. But these methods are useless.
I suggest you can set these method as "optional" in "BitcoinSigner" interface. So you don't have to write such methods in mockBtc.ts
About optional method in typescript. I find https://compiletab.com/optional-properties-in-interfaces-typescript/

Reply:
As seen together, we need to define those methods to conform to the interface definition (a.k.a. `BitcoinSigner`)